### PR TITLE
Update contributing docs. Clarification about validation-online

### PR DESCRIPTION
**Pull request by luisprmat**
I added some clarification in contributing docs about `validation-online.php`

### CODE REVIEW

File: docs/contributing-to-dev.md [hunk 1]
```diff
@@ -11,7 +11,13 @@ We accept contributions via Pull Requests on [Github](https://github.com/Laravel
...
 * create a directory in *src* with the short name of the language (ex: fr for French) from ISO-639-1 (
   see [Wikipedia](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) )
-* copy these four files of the English version with your translation : from `script/en` directory
+* copy these five files of the English version with your translation : from `script/en` directory
+    * `auth.php`
+    * `pagination.php`
+    * `passwords.php`
+    * `validation.php`
+    * `validation-inline.php`
+* Keep in mind that the `validation-online.php` file does not come with Laravel and the idea of ​​this file is not to put a specific name to each attribute (as in `validation.php`) but a generic name for the validation attributes. Therefore in the translations of this file the placeholder `:attribute` **should not** appear.
```
Comment (caouecs) [inline docs/contributing-to-dev.md:20]: validation-online.php => validation-inline.php

**Review by caouecs — CHANGES_REQUESTED** [review]
gracias, pero hay un error

File: docs/contributing-to-dev.md (hunk 1 above)
Comment (luisprmat) [reply docs/contributing-to-dev.md]: Excuse me for typo. Fixed!

File: docs/contributing-to-dev.md [hunk 2]
```diff
@@ -11,7 +11,13 @@ We accept contributions via Pull Requests on [Github](https://github.com/Laravel
...
 * create a directory in *src* with the short name of the language (ex: fr for French) from ISO-639-1 (
   see [Wikipedia](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) )
-* copy these four files of the English version with your translation : from `script/en` directory
+* copy these five files of the English version with your translation : from `script/en` directory
+    * `auth.php`
+    * `pagination.php`
+    * `passwords.php`
+    * `validation.php`
+    * `validation-inline.php`
+* Keep in mind that the `validation-inline.php` file does not come with Laravel and the idea of ​​this file is not to put a specific name to each attribute (as in `validation.php`) but a generic name for the validation attributes. Therefore in the translations of this file the placeholder `:attribute` **should not** appear.
```
Comment (caouecs) [inline docs/contributing-to-dev.md:20]: Keep => keep

**Review by caouecs — CHANGES_REQUESTED** [review]
sorry :)

**Comment by luisprmat** [comment]
@andrey-helldar It would also be nice to add an *assert* to `testValidationOnline` so that the tests do not pass if the string `:attribute` appears in `validation-online.php`. I guess something like
```php
$this->assertDoesntSee('validation-online.php', ':attribute');
```
for every language

File: docs/contributing-to-dev.md (hunk 1 above)
Comment (andrey-helldar) [reply docs/contributing-to-dev.md]: validation-offline 😀

**Comment by caouecs** [comment]
> @andrey-helldar It would also be nice to add an _assert_ to `testValidationOnline` so that the tests do not pass if the string `:attribute` appears in `validation-online.php`. I guess something like
> 
> ```
> $this->assertDoesntSee('validation-online.php', ':attribute');
> ```
> 
> for every language

validation-inline.php ;)

**Comment by caouecs** [comment]
Grasias

**Comment by andrey-helldar** [comment]
> @andrey-helldar It would also be nice to add an _assert_ to `testValidationOnline` so that the tests do not pass if the string `:attribute` appears in `validation-online.php`. I guess something like
> 
> ```
> $this->assertDoesntSee('validation-online.php', ':attribute');
> ```
> 
> for every language

I agree. I'll add this test today :)

**Pull request merged**